### PR TITLE
Fix parent job id

### PR DESCRIPTION
**Pull request by bio-boris**
Error while trying to start your app (run_app_bulk)!
-----------------------------------------------------
Unable to fetch info for job 60d6205d09b619fd9d292ad6 - Server error: -32000. "'NoneType' object has no attribute 'parent_job_id'"
Traceback (most recent call last):
  File "/kb/module/lib/execution_engine2/execution_engine2Server.py", line 101, in _call_method
    result = method(ctx, *params)
  File "/kb/module/lib/execution_engine2/execution_engine2Impl.py", line 1011, in check_job
    job_state = mr.check_job(
  File "/kb/module/lib/execution_engine2/sdk/SDKMethodRunner.py", line 418, in check_job
    return self.get_jobs_status().check_job(
  File "/kb/module/lib/execution_engine2/sdk/EE2Status.py", line 419, in check_job
    job_state = self.check_jobs(
  File "/kb/module/lib/execution_engine2/sdk/EE2Status.py", line 477, in check_jobs
    mongo_rec["parent_job_id"] = job.job_input.parent_job_id
AttributeError: 'NoneType' object has no attribute 'parent_job_id'

-----------------------------------------------------
Traceback (most recent call last):
  File "/kb/runtime/lib/python3.6/site-packages/biokbase-0.0.1-py3.6.egg/biokbase/narrative/jobs/job.py", line 170, in state
    {"job_id": self.job_id, "exclude_fields": EXCLUDED_JOB_STATE_FIELDS}
  File "/kb/runtime/lib/python3.6/site-packages/biokbase-0.0.1-py3.6.egg/biokbase/execution_engine2/execution_engine2Client.py", line 730, in check_job
    "execution_engine2.check_job", [params], self._service_ver, context
  File "/kb/runtime/lib/python3.6/site-packages/biokbase-0.0.1-py3.6.egg/biokbase/execution_engine2/baseclient.py", line 296, in call_method
    return self._call(url, service_method, args, context)
  File "/kb/runtime/lib/python3.6/site-packages/biokbase-0.0.1-py3.6.egg/biokbase/execution_engine2/baseclient.py", line 212, in _call
    raise ServerError(**err["error"])
biokbase.execution_engine2.baseclient.ServerError: Server error: -32000. "'NoneType' object has no attribute 'parent_job_id'"
Traceback (most recent call last):
  File "/kb/module/lib/execution_engine2/execution_engine2Server.py", line 101, in _call_method
    result = method(ctx, *params)
  File "/kb/module/lib/execution_engine2/execution_engine2Impl.py", line 1011, in check_job
    job_state = mr.check_job(
  File "/kb/module/lib/execution_engine2/sdk/SDKMethodRunner.py", line 418, in check_job
    return self.get_jobs_status().check_job(
  File "/kb/module/lib/execution_engine2/sdk/EE2Status.py", line 419, in check_job
    job_state = self.check_jobs(
  File "/kb/module/lib/execution_engine2/sdk/EE2Status.py", line 477, in check_jobs
    mongo_rec["parent_job_id"] = job.job_input.parent_job_id
AttributeError: 'NoneType' object has no attribute 'parent_job_id'

### CODE REVIEW

File: lib/execution_engine2/sdk/EE2Status.py [hunk 1]
```diff
@@ -474,7 +474,7 @@ def check_jobs(
                 del mongo_rec["_id"]
                 mongo_rec["retry_count"] = len(job["retry_ids"])
                 mongo_rec["job_id"] = str(job.id)
-                mongo_rec["parent_job_id"] = job.job_input.parent_job_id
+                mongo_rec["parent_job_id"] = job.job_input.parent_job_id if job_job.input else None
```
Comment (MrCreosote) [inline lib/execution_engine2/sdk/EE2Status.py:477]: How can `job_input` ever be `None`?

Comment (MrCreosote) [reply lib/execution_engine2/sdk/EE2Status.py:477]: Also looks like there's a switched _ and . in job_job.input

Comment (bio-boris) [reply lib/execution_engine2/sdk/EE2Status.py:477]: For parent_jobs, there is no job input

**Comment by MrCreosote** [comment]
Well, other than the tests failing

**Pull request merged**